### PR TITLE
add case-import-server deployment

### PR DESCRIPTION
**Pull request by AbdelHedhili**

### CODE REVIEW

File: k8s/resources/study/kustomization.yaml [hunk 1]
```diff
@@ -118,5 +120,10 @@ configMapGenerator:
   - name: voltage-init-server-itools-configmap
     files:
       - config.yml=config/voltage-init-server-config.yml
-
+  - name: voltage-init-server-itools-configmap
```
Comment (FranckLecuyer) [inline k8s/resources/study/kustomization.yaml:123]: duplicated

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Missing case-import-server in main.yml file
Missing case-import-server in README.md

**Comment by AbdelHedhili** [comment]
> Maybe add an empty case-import-server-application.yml to be homogeneous with the other services

We don't seem to have these empty files for all the servers, I'm not even sure why they exist, but I added it

File: k8s/resources/study/kustomization.yaml (hunk 1 above)
Comment (AbdelHedhili) [reply k8s/resources/study/kustomization.yaml]: fixed

**Review by FranckLecuyer — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**